docs/operational_guide/namespace_configuration.md

```diff
@@ -128,6 +128,16 @@ This is the most important value to consider when tuning the performance of an M
 
 Can be modified without creating a new namespace: `no`
 
+Below are recommendations for block size based on the resolution and retention of the namespace:
+
+| Retention / Resolution | 10s | 1m  | 10m | 1h  |
+|------------------------|-----|-----|-----|-----|
+| 12h                    | 30m | 2h  | N/A | N/A |
+| 24h                    | 1h  | 4h  | N/A | N/A |
+| 168h                   | 6h  | 12h | 24h | 48h |
+| 720h                   | 24h | 24h | 48h | 96h |
+| 8760h                  | 24h | 24h | 48h | 96h |
```

Comment:
We should use a snippet instead of repeating this content.

See the following example (search for where this is used to see how to pull in from the `common` directory):
https://github.com/m3db/m3/blob/master/docs/common/headers_optional_read_write.md